src/main/java/com/flowingcode/addons/ycalendar/DatePickerEx.java

```diff
@@ -0,0 +1,99 @@
+/*-
+ * #%L
+ * Year Month Calendar Add-on
+ * %%
+ * Copyright (C) 2021 - 2025 Flowing Code
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package com.flowingcode.addons.ycalendar;
+
+import com.vaadin.flow.component.ClientCallable;
+import com.vaadin.flow.component.Tag;
+import com.vaadin.flow.component.datepicker.DatePicker;
+import com.vaadin.flow.component.dependency.JsModule;
+import com.vaadin.flow.function.ValueProvider;
+import elemental.json.Json;
+import elemental.json.JsonObject;
+import java.time.LocalDate;
+import java.time.YearMonth;
+import java.util.Optional;
+
+@SuppressWarnings("serial")
+@Tag("fc-date-picker")
+@JsModule("./fc-date-picker/fc-date-picker.js")
+public class DatePickerEx extends DatePicker {
```

Comment:
What the "Ex" stands for in the name? It's confusing for me. It is "Ex" for "extended"? If it's shorten for something else wouldn't be better to use the whole word?

Reply:
+1 I would be more explicit with the naming, maybe something like ExtendedDatePicker or similar (if Ex is a shorter form of Extended)